.github/workflows/tests.yml

```diff
@@ -8,11 +8,21 @@ on:
 
 jobs:
   test:
-    name: Run Unit Tests
+    name: Run Unit Tests PHP ${{ matrix.php }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '8.1', '8.2', '8.3' ]
     steps:
     - uses: actions/checkout@v2
 
+    -  name: Installing PHP
```

Comment:
"Installing" should be "Install" and I would include the version in the name, so:

```
- name: Install PHP ${{ matrix.php }}

Reply:
OK, changes arriving shortly